src/sentry/utils/snuba.py

```diff
@@ -1283,3 +1296,16 @@ def quantize_time(time, key_hash, duration=300):
         # Use timedelta here so keys are consistent around hour boundaries
         timedelta(seconds=seconds_past_hour)
     )
+
+
+def is_measurement(key):
+    return isinstance(key, six.string_types) and MEASUREMENTS_KEY_RE.match(key)
+
+
+def is_duration_measurement(key):
+    return is_measurement(key) and key in [
```

Comment:
Do you need to call is_measurement if you're also checking the membership of the allowed measures?

Reply:
No, it's not necessary, I'll remove it.

Reply:
Addressed in 2f7d0a1